config/routes/old_route_redirects.rb

```diff
@@ -19,8 +19,9 @@
       to: redirect('/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/%{id}')
 
   # redirect old js pathroutes
-  get '/courses/nodeJS', to: redirect('/paths/full-stack-javascript/courses/nodeJS')
-  get '/courses/nodeJS/lessons/:id', to: redirect('/paths/full-stack-javascript/courses/nodeJS/lessons/%{id}')
+  get '/courses/nodeJS', to: redirect('/paths/full-stack-javascript/courses/nodejs')
+  get '/courses/nodejs', to: redirect('/paths/full-stack-javascript/courses/nodejs')
+  get '/courses/nodeJS/lessons/:id', to: redirect('/paths/full-stack-javascript/courses/nodejs/lessons/%{id}')
```

Comment:
Hey @ZachBaird, thanks for doing this. This is my fault 🙈  the course name has that capitalisation in the seeds  but rails turns it to lower case with the routes.

We should be able to remove line 22 and change line 24's url to be redirected to use the lowercase nodejs as well.

Reply:
Say no more mon ami, it is done!